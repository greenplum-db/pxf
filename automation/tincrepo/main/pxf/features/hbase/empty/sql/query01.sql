-- @description query01 for PXF HBase empty table

SELECT * from empty_hbase_table;