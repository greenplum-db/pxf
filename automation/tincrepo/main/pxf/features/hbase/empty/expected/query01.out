-- @description query01 for PXF HBase empty table

SELECT * from empty_hbase_table;
 recordkey | cf1:q1 | cf1:q2 | cf1:q3 
-----------+--------+--------+--------
(0 rows)
